harvester_e2e_tests/apis/test_volumes.py

```diff
@@ -119,3 +153,48 @@ def test_delete(self, api_client, unique_name, wait_timeout):
                 f"Failed to delete volume {unique_name} with {wait_timeout} timed out\n"
                 f"Still got {code} with {data}"
             )
+
+    def test_create_volume_backing_image(self, request, api_client, unique_name):
```

Comment:
as this test case requires external resource, we should move it to `integration`

Reply:
removed request and just used the images fixture

Reply:
Were you referring to the part? I removed that, but it just occurred to me that you might be referring to the external image (in this case opensuse)?

Reply:
moved.